ember.js/2016-08/august-19.md

```diff
@@ -0,0 +1,248 @@
+# Ember Core Team 2016-08-19
+
+### Agenda
+
+- (LS) Updated Getting Started Video to replace outdated one, any comments or thoughts?
+- (SP / RJ) [RFC for Hook/Event for Transitioning During a Transition](https://github.com/emberjs/rfcs/pull/126) --- Propose final comment period... 
+- (RJ) Enabling ember-glimmer feature flag for canary. (public)
+
+Blockers for rolling to beta:
+
+             - [ ] Migrating & fixing all remaining skipped tests.
+             - [ ] Review and comment on all open "meta" issues.
+             - [ ] ???
+
+### Attendees
+
+AM, DG, EF, GC, LS, RJ, TD, MB, YK
+
+### Minutes
+
+#### Getting Stated Video
+
+YK: Seems good, Gavin is great and this is based on the github app
+
+MB: Uses ember-data github
+
+YK: Yeah that seems less than ideal
+
+TD: the ideal would be to have the intro video not use ember-data at all, and have a second one that migrates the code to ember-data
+
+TD: #millenialshaming, we should optimize for a short video format
+
+YK: DHH's 15 minutes video is not compatible with the way we share and talk about things to today. I agree.
+
+TD: They are also linkable, each section would be "I want to think about this"
+
+YK: If you want to make smaller videos you need to think about what discrete chunk of knowledge you're teaching
+
+RJ: Unless I missed it, also this does not talk about tests
+
+YK: It is hard to do tests in 15 minutes
+
+TD: If you chunk it up, you do it as build the feature and test it.
+
+YK: I think if the chunks get too long (10m) it seems difficult
+
+GC: Five minutes sounds better
+
+TD: The marketing message angular had with testing was really powerful early on. We can probably use that to our advantage now especially if we have the improved testing items.
+
+GC: Anything we can do to keep it up to date?
+
+YK: For me screenflow has been the only thing useful. I'd like a tool that let you script this. Applescript can do stuff like this.
+
+YK: If you restrict it you could likely build something. Making it generic is what is the pain in an ass.
+
+#### RFC for transition event
+
+RJ: I'd like to FCP this
+
+YK: I have some blocking thoughts.
+
+(discussion)
+
+YK: There are several hooks in that space, and I'm hesitant to add another one without understanding the impact on teaching, or knowing how to steer people away from it when they shouldn't land there.
+
+KS: It overlaps with the routing service right?
+
+EF: There are things in this RFC that take private objects and makes them effectively public. They would also need to be documented and thought about.
```

Comment:
The `willTransition` hook receives exactly the same set of arguments as the proposed new hook. There is no additional API surface area.